src/Avalonia.Controls/Platform/MacOSProperties.cs

```diff
@@ -32,4 +35,34 @@ private static void TrayIconIsTemplateIconChanged(TrayIcon trayIcon, AvaloniaPro
     {
         (trayIcon.Impl as ITrayIconWithIsTemplateImpl)?.SetIsTemplateIcon(args.GetNewValue<bool>());
     }
+
+    /// <summary>
+    /// Defines the CanBecomeKeyWindow attached property.
+    /// </summary>
+    public static readonly AttachedProperty<bool> CanBecomeKeyWindowProperty =
```

Comment:
While it's the term used in the macOS API, "key window" isn't used in Avalonia at all. It might not be clear that _key_ here refers to keyboard input, as opposed to an "important" window.

I think something with _focus_ in the name would be better, such as `IsNativeFocusable` or `CanGetNativeFocus` (we recently had `Popup.TakesFocusFromNativeControl` merged).

What do you think?

Reply:
The majority of the team feels like using the OS-specific name is the way to go since it's an OS-specific property. The name stays `CanBecomeKeyWindow`.